# Basic ROS message for sensor data packet
Header header
string data_type
uint64 sequence_id
uint8[] payload